### PR TITLE
Rename the logical operation "not" to "logicalNot"

### PR DESCRIPTION
**Pull request by shiyi9801**
As per discussion in #496, use name "logicalNot" to reduce ambiguity.

***
<a href="https://pr-preview.s3.amazonaws.com/shiyi9801/webnn/pull/692.html" title="Last updated on May 24, 2024, 5:55 AM UTC (c852d7f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/webnn/692/960ae89...shiyi9801:c852d7f.html" title="Last updated on May 24, 2024, 5:55 AM UTC (c852d7f)">Diff</a>

### CODE REVIEW

**Review by huningxin — APPROVED** [review]
LGTM, thanks!

**Review by huningxin — APPROVED** [review]
LGTM, thanks for the fix.

**Comment by anssiko** [comment]
@shiyi9801 thanks for the contribution. I have cleared the CI check for you.

This automated CI check is in place to ensure contributions containing so-called substantive changes come from WG participants only. Since this change is a rename it is considered non-substantive from the IPR point of view. Frequent contributors who are planning to contribute substantive changes are expected to join the WG.

**Review by fdwr — APPROVED** [review]
Thanks Shiyi. Although the name is longer now, it adds clarity to the future family (and aligns the implementation with spec).

@anssiko Should we wait until the W3C meeting to merge it, or is merging sooner fine?

**Comment by anssiko** [comment]
@fdwr @huningxin, we did not have time to discuss this in our meeting today. If you editors feel some individuals should be consulted prior to merge please tag them here. If this is considered straightforward change and you are confident you can manage the impact of this rename on the layers above, then you can merge. I trust in your judgment on this matter.

@shiyi9801 if you plan on making substantive contributions in this WG, my expectation is you will join the group. Alternatively you can let Intel's current participants submit the PRs with credits to you.

**Comment by huningxin** [comment]
> @fdwr @huningxin, we did not have time to discuss this in our meeting today. If you editors feel some individuals should be consulted prior to merge please tag them here. If this is considered straightforward change and you are confident you can manage the impact of this rename on the layers above, then you can merge. I trust in your judgment on this matter.

I think this PR got 3 approvals and is good to merge.

> @shiyi9801 if you plan on making substantive contributions in this WG, my expectation is you will join the group. Alternatively you can let Intel's current participants submit the PRs with credits to you.

Thanks for the guidance. I can coordinate with Shiyi for other potential PRs.

**Pull request merged**
